expression/builtin_string_test.go

```diff
@@ -2226,21 +2226,27 @@ func TestOrd(t *testing.T) {
 	cases := []struct {
 		args     interface{}
 		expected int64
+		chs      string
 		isNil    bool
 		getErr   bool
 	}{
-		{"2", 50, false, false},
-		{2, 50, false, false},
-		{"23", 50, false, false},
-		{23, 50, false, false},
-		{2.3, 50, false, false},
-		{nil, 0, true, false},
-		{"", 0, false, false},
-		{"你好", 14990752, false, false},
-		{"にほん", 14909867, false, false},
-		{"한국", 15570332, false, false},
-		{"👍", 4036989325, false, false},
-		{"א", 55184, false, false},
+		{"2", 50, "", false, false},
+		{2, 50, "", false, false},
+		{"23", 50, "", false, false},
+		{23, 50, "", false, false},
+		{2.3, 50, "", false, false},
+		{nil, 0, "", true, false},
+		{"", 0, "", false, false},
+		{"你好", 14990752, "", false, false},
+		{"にほん", 14909867, "", false, false},
+		{"한국", 15570332, "", false, false},
+		{"👍", 4036989325, "", false, false},
+		{"א", 55184, "", false, false},
+		// TODO: Uncomment it when gbk be added into charsetInfos
```

Comment:
Please use `SetCharsetFeatEnabledForTest` to enable the new-character-set feature.

Reply:
Resolved.